slacktee.sh

```diff
@@ -193,22 +192,41 @@ function send_message()
 
 		username=$(escape_string "$username")
 
-		json="{\
-                  \"channel\": \"$channel\", \
-                  \"username\": \"$username\", \
-                  $message_attr \"icon_emoji\": \"$icon_emoji\", \
-                  \"icon_url\": \"$icon_url\" $parseMode}"
-		post_result=$(curl -X POST --data-urlencode \
-                  "payload=$json" "$webhook_url" 2> /dev/null)
-                if [[ $post_result != "ok" ]]; then
-                	err_exit 1 "$post_result"
-                fi
+		if [[ $mode == "streaming" ]]; then
+			if [[ -z "$streaming_ts" ]]; then
+				post_result=$(curl -d "token=$token&username=$username&icon_url=$icon_url&icon_emoji=$icon_emoji&$parseModeUrlEncoded&channel=$channel&text=$wrapped_message" -X POST https://slack.com/api/chat.postMessage 2> /dev/null)
+				if [ $? != 0 ]; then
+			    err_exit 1 "$post_result"
```

Comment:
Please fix the indentation.

Reply:
oops, I mixed some spaces with your tabs. fixed